share/completions/magento.fish

```diff
@@ -0,0 +1,720 @@
+__fish_magento_register_command_option admin:user:create -f -s q -l quiet          -d "Do not output any message";
+__fish_magento_register_command_option admin:user:create -f -s V                   -d "Display this application version";
+__fish_magento_register_command_option admin:user:create -f      -l ansi           -d "Force ANSI output";
+__fish_magento_register_command_option admin:user:create -f      -l no-ansi        -d "Disable ANSI output";
+__fish_magento_register_command_option admin:user:create -f -s n -l no-interaction -d "Do not ask any interactive question";
+__fish_magento_register_command_option admin:user:create -f -s v -l verbose -a "(__fish_print_magento_verbosity_levels)" -d "Increase the verbosity of messages: 1 for normal output, 2 for more verbose output and 3 for debug";
+
+#
+# admin:user:unlock
+#
+__fish_magento_register_command_option admin:user:unlock -f -a "(__fish_print_magento_users)" -d "The admin username to unlock";
+
+#
+# cache:clean
+#
+__fish_magento_register_command_option cache:clean -f -a "(__fish_print_magento_cache_types)" -d "Space-separated list of cache types or omit to apply to all cache types.";
+__fish_magento_register_command_option cache:clean -f -l bootstrap -d "add or override parameters of the bootstrap";
+
+#
+# cache:enable
+#
+__fish_magento_register_command_option cache:enable -f -a "(__fish_print_magento_cache_types)" -d "Space-separated list of cache types or omit to apply to all cache types.";
+__fish_magento_register_command_option cache:enable -f -l bootstrap -d "add or override parameters of the bootstrap";
+
+#
+# cache:disable
+#
+__fish_magento_register_command_option cache:disable -f -a "(__fish_print_magento_cache_types)" -d "Space-separated list of cache types or omit to apply to all cache types.";
+__fish_magento_register_command_option cache:disable -f -l bootstrap -d "add or override parameters of the bootstrap";
+
+#
+# cache:status
+#
+__fish_magento_register_command_option cache:status -f -l bootstrap -d "add or override parameters of the bootstrap";
+
+#
+# cron:run
+#
+__fish_magento_register_command_option cron:run -f -l group -a "TODO" -d "Run jobs only from specified group";
+__fish_magento_register_command_option cron:run -f -l bootstrap -d "add or override parameters of the bootstrap";
+
+#
+# deploy:mode:set
+#
+__fish_magento_register_command_option deploy:mode:set -f -a "(__fish_print_magento_deploy_modes)" -d 'The application mode to set. Available options are "developer" or "production"';
+__fish_magento_register_command_option deploy:mode:set -f -s s -l skip-compilation -d "Skips the clearing and regeneration of static content (generated code, preprocessed CSS, and assets in pub/static/)";
+
+#
+# dev:source-theme:deploy
+#
+__fish_magento_register_command_option dev:source-theme:deploy -f -a "" -d 'Files to pre-process (file should be specified without extension) (default: ["css/styles-m","css/styles-l"])';
```

Comment:
Try to keep descriptions short - 'Files to preprocess (without extension)'. Ideally the default would be mentioned when it comes to the option.